README.md

```diff
@@ -1,2 +1,242 @@
-# module-ballerinax-microsoft.sheets
-The Ballerina connector to perform operations on Excel workbooks stored in Microsoft OneDrive.
+# Module Microsoft Sheets
+
+The `ballerinax/module-ballerinax-microsoft.sheets` module contains operations to perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | Swan Lake Preview1   |
+| Microsoftgraph REST API | v1.0          |
+
+## Getting started
+
+1.  Download and install Ballerina. For instructions, go to [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/).
+
+2.  Provide the following configuration information in the `ballerina.conf` file to use the Microsoft Graph API.
+
+       - MS_CLIENT_ID
+       - MS_CLIENT_SECRET
+       - MS_ACCESS_TOKEN
+       - MS_REFRESH_TOKEN
+       - TRUST_STORE_PATH
+       - TRUST_STORE_PASSWORD
+       - WORK_BOOK_NAME
+       - WORK_SHEET_NAME
+       - TABLE_NAME
+
+    Follow the steps below to obtain the configuration information mentioned above.
+
+    Before you run the following steps, create an account in [OneDrive](https://onedrive.live.com). Next, sign into [Azure Portal - App Registrations](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade). You can use your personal or work or school account to register.
+
+    In the App registrations page, click **New registration** and enter a meaningful name in the name field.
+
+    !["Figure 1: App registrations page"](images/step1.jpg)
+
+    *Figure 1: App registrations page*
```

Comment:
```suggestion
    *Figure 1: App registrations page*
```
It is not needed to add captions for images.

Reply:
Done